.circleci/create_circleci_config.py

```diff
@@ -472,6 +472,7 @@ def job_name(self):
         "pip install -U --upgrade-strategy eager 'git+https://github.com/facebookresearch/detectron2.git'",
         "sudo apt install tesseract-ocr",
         "pip install -U --upgrade-strategy eager pytesseract",
+        "pip install --upgrade-strategy eager sentencepiece",
```

Comment:
this should not be added no?

Reply:
UDOP leverages SentencePiece for its tokenizer, hence this would be required as UDOP is part of the exotic model tests

Reply:
ah ok understood